ngraph/frontend/paddlepaddle/src/op/range.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
+#include <node_context.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs range(const NodeContext& node)
+                {
+                    auto start = node.get_ng_input("Start");
+                    auto stop = node.get_ng_input("End");
+                    auto step = node.get_ng_input("Step");
+                    auto type = node.get_out_port_type("Out");
+                    PDPD_ASSERT(type == element::i64 || type == element::i32 ||
+                                    type == element::f32,
+                                "Only supports int32, int64, float32");
```

Comment:
Will this assert print out the node name and type? Seems like user would no nothing what he/she can do with this.

Reply:
Also, why we check it here? If Range doesn't support some type -- Range `validate_and_infer_type' would verbosely report it Please get rid of this check.

Reply:
Previously, we want to highlight that only those types are supported. But you are right we can remove this since validate_and_infer_type already have these check